fe-ionic/files/package.json

```diff
@@ -1,8 +1,6 @@
 {
-  "name": "ionic-quickstart",
-  "version": "0.0.1",
-  "author": "BIX",
-  "private": true,
```

Comment:
Any reason you removed this ("private": true)?
Is no big deal, but this avoids an accidental publish to a package repository.
I am OK if we say this in the developers responsibility, just to make sure we want this (since we normally do not publish our full apps to a npm repo)...

Reply:
I was just trying to be compliant to other fe repository, like fe-angular. Maybe we should keep this private setting

Reply:
Then we just found something we can add to all other node based Quickstarters I guess? 😅